WcaOnRails/db/migrate/20220627195217_add_allow_registration_without_qualification_to_competitions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddAllowRegistrationWithoutQualificationToCompetitions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :Competitions, :allow_registration_without_qualification, :boolean, default: true
```

Comment:
What's the rationale behind using `true` as default?

Reply:
I think you should be able to do this as part of the current migration. Something along the lines of
```ruby
def change
  # N.B. This has the default set to false
  add_column ...foo... default: false
  
  # Except all of the competitions that are currently open
  Competition.where("registration_close >= :close", close: DateTime.now).update_all(allow_registration_without_qualification: true)
end
```

Reply:
Done!